### PR TITLE
open issues on repo when managers fail

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

#726

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

If managers fails then an issue along with a trace will be created on the users repository so that they can either open an upstream issue with us or fix their repository.

### CODE REVIEW

**Comment by KPostOffice** [comment]
/hold

I tested this and the basic functionality is working, but I would like to add surrounding text 
to make it more actionable to the end-user.

**Review by harshad16 — COMMENTED** [review]
lgtm
thank, fix the precommit issue :+1:

**Comment by KPostOffice** [comment]
/unhold
See here for an example 
https://github.com/KPostOffice/jupyterlab-requirements/issues/6

**Comment by KPostOffice** [comment]
@harshad16 PTAL

File: kebechet/managers/manager.py [hunk 1]
```diff
@@ -31,6 +31,8 @@
 from ogr.services.base import BaseGitService
 from ogr.abstract import Issue, PullRequest
 
+from kebechet import utils
```
Comment (harshad16) [inline kebechet/managers/manager.py:34]: this is not stated in __init__.py , how do this work ?
should this be , from kebechet.utils import get_issue_by_title .

**Review by harshad16 — COMMENTED** [review]
one minor doubt @KPostOffice

**Comment by KPostOffice** [comment]
@harshad16 PTAL

File: kebechet/managers/manager.py (hunk 1 above)
Comment (KPostOffice) [reply kebechet/managers/manager.py:34]: This works fine, utils should be discoverable as a module by file name alone. I left it as `import utils` so the distinction between `self.get_issue_by_title` and `utils.get_issue_by_title` was more clear.

**Review by harshad16 — APPROVED** [review]
thanks :100:

**Pull request merged**